release/airflow/cloud_build.template.json

```diff
@@ -32,7 +32,7 @@
       ],
       "name": "gcr.io/istio-testing/istio-builder:0.5.10",
       "dir": "go/src/istio.io/istio",
-      "args": [ "./release/store_artifacts.sh", "-p", "$_GCS_PATH", "-q", "$_GCR_PATH", "-v", "$_VER_STRING", "-o", "/output" ]
+      "args": [ "./release/store_artifacts.sh", "-c", "$_BRANCH", "-p", "$_GCS_PATH", "-h", "$_DOCKER_HUB", "-v", "$_VER_STRING", "-o", "/output" ]
```

Comment:
why is GCR_PATH removed in most of these files?

I suppose we still publish to GCR (along with pushing to dovkrthub), Just that istioctl hub is defaulted to dockerhub, right?

Reply:
If DOCKER_HUB can be GCR, should it be named as something like REGISTRY_PATH?